src/data.py

```diff
@@ -133,14 +133,21 @@ def calculate_natural_breaks_bins(df: pd.DataFrame,
     return bin_dict
 
 
-def calculate_quantiles_bins(df: pd.DataFrame, columns_to_bin: List[str]) -> Dict[str, List[float]]:  # noqa: E501
+def make_quantiles_q(bin_count: int) -> List[float]:
+    return [round(index / bin_count, 2)
+            for index in range(0, bin_count + 1)]
```

Comment:
The sum of the quantiles will always be 1 right?

Should we add a check?

Reply:
I'm satisfied enough with the unit test to think we have it correct:


<!--StartFragment-->def test_make_quantiles_q():    assert data.make_quantiles_q(2) == [0, 0.5, 1]    assert data.make_quantiles_q(3) == [0, 0.33, 0.67, 1]    assert data.make_quantiles_q(4) == [0, 0.25, 0.5, 0.75, 1]    assert data.make_quantiles_q(5) == [0, 0.2, 0.4, 0.6, 0.8, 1]    assert data.make_quantiles_q(10) == [0, 0.1, 0.2, 0.3, 0.4, 0.5,                                         0.6, 0.7, 0.8, 0.9, 1]<!--EndFragment-->